api/files/views.py

```diff
@@ -34,11 +34,13 @@ class FileMixin(object):
     serializer_class = FileSerializer
     file_lookup_url_kwarg = 'file_id'
 
-    def get_file(self, check_permissions=True):
+    def get_file(self, check_permissions=True, ser=False):
```

Comment:
A more descriptive argument name here (not an abbreviation) might be better. At first glance I would assume `ser` is short for `serializer`, but that makes me think that a serializer class would be passed, not a boolean value.